boot/nuttx/main.c

```diff
@@ -79,17 +80,19 @@ int main(int argc, FAR char *argv[])
   struct boot_rsp rsp;
   fih_int fih_rc = FIH_FAILURE;
 
-  printf("*** Booting MCUboot build %s ***\n", CONFIG_MCUBOOT_VERSION);
+  syslog(LOG_INFO, "*** Booting MCUboot build %s ***\n", CONFIG_MCUBOOT_VERSION);
 
   FIH_CALL(boot_go, fih_rc, &rsp);
 
   if (fih_not_eq(fih_rc, FIH_SUCCESS))
     {
-      fprintf(stderr, "Unable to find bootable image\n");
+      syslog(LOG_ERR, "Unable to find bootable image\n");
       FIH_PANIC;
     }
 
   do_boot(&rsp);
 
+  syslog(LOG_ERR, "Something went wrong, should not get here!");
```

Comment:
This log line adds little to no information to the Bootloader user.
I suggest to either rephrase it by adding a proper error message with context information, or simply remove it.

Reply:
If the intent is to catch a missing (or bad) implementation of `BOARDIOC_BOOT_IMAGE` boardctl call, then catch this error on the `do_boot` function.